core/internal/cltest/cltest.go

```diff
@@ -199,33 +199,20 @@ func NewJobPipelineV2(t testing.TB, cfg config.BasicConfig, cc evm.ChainSet, db
 	}
 }
 
-// NewEthBroadcaster creates a new txmgr.EthBroadcaster for use in testing.
-func NewEthBroadcaster(t testing.TB, orm txmgr.ORM, ethClient evmclient.Client, keyStore keystore.Eth, config evmconfig.ChainScopedConfig, addresses []common.Address, checkerFactory txmgr.TransmitCheckerFactory, nonceAutoSync bool) *txmgr.EthBroadcaster {
```

Comment:
I don't fully recall, but I believe ever since we do "DisableUnstartedEthTxAutoProcessing" on that eb instance for unit-tests, it shouldn't be mistakenly set by any other code. So I moved it in the test file here.
Any other test code which wants to use ethBroadcaster, should not use this method.